main.c

```diff
@@ -20,12 +21,23 @@ char		**ft_trim_user_input(char **argv, int argc);
 void		print_tokens(t_token *list);
 void		minishell_interactive(t_global *global);
 
+
+
+
+// NOTE: comment this function to run test cases from tests.c
```

Comment:
I understand this is done to work in 'test moded' on our minishell - let's discuss creating a new `Makefile` rule: `tests` and having the criterion library working there
Otherwise we might run into a scenario when in the end we uncomment our proper `production` main and things stop working because we were not updating our main along the way
Happy to discuss further - this is not a big issue for now so if convinced we can also go with this version

Reply:
Let's leave it for now as it it for the purpose of merging this branch before tomorrow :)
We can discuss tomorrow